packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -1259,6 +1446,15 @@ interface RebaseTable extends CrossFieldTable<FieldChange> {
 	 * changes. Instead we want to keep using the objects generated in the first pass and mutate them where needed.
 	 */
 	rebasedNodeCache: Map<NodeChangeset, NodeChangeset>;
+
+	/**
+	 * List of (newId, baseId) pairs encountered so far.
+	 */
+	nodeIdPairs: [
+		NodeId | undefined,
+		TaggedChange<NodeId> | undefined,
```

Comment:
I'm a little confused by `TaggedChange<NodeId>`: is there ever a case where `tagged.change.revision` woud be different from `tagged.revision`?

Reply:
It is possible when the node ID has an undefined revision, although in that case it does still implicitly have the revision of the parent node.